<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAPASITY_SHIPS_Dialog</class>
 <widget class="QDialog" name="CAPASITY_SHIPS_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#CAPASITY_SHIPS_Dialog {
	background-color: rgb(187, 245, 243);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="90,10">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="25,75">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Id:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
border: 1px solid #000000;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="25,75">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Тип груза:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
     width: 10px;
	background-color:  rgb(0, 230, 245);
    image: url(:/new/prefix1/1.png);
    padding-left: 7px;
	padding-right: 7px;

    border-left-width: 1px;
    border-left-color: black;
    border-left-style: solid;  
 }
QComboBox {
background-color: rgb(255, 255, 255);
border: 1px solid black;
}
QComboBox QAbstractItemView {
    background-color: rgb(255, 255, 255);
 }</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="25,75">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Количество:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
border: 1px solid #000000;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="25,75">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Судно:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
     width: 10px;
	background-color:  rgb(0, 230, 245);
    image: url(:/new/prefix1/1.png);
    padding-left: 7px;
	padding-right: 7px;

    border-left-width: 1px;
    border-left-color: black;
    border-left-style: solid;  
 }
QComboBox {
background-color: rgb(255, 255, 255);
border: 1px solid black;
}
QComboBox QAbstractItemView {
    background-color: rgb(255, 255, 255);
 }</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="20,40,20,20">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(0, 230, 245);
font:bold;
border: 1px solid #000000;
}
QPushButton::hover {
	background-color: rgb(0, 243, 255);
}
QPushButton::pressed {
	border: 3px solid #000000;
}</string>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(0, 230, 245);
font:bold;
border: 1px solid #000000;
}
QPushButton::hover {
	background-color: rgb(0, 243, 255);
}
QPushButton::pressed {
	border: 3px solid #000000;
}</string>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(0, 230, 245);
font:bold;
border: 1px solid #000000;
}
QPushButton::hover {
	background-color: rgb(0, 243, 255);
}
QPushButton::pressed {
	border: 3px solid #000000;
}</string>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
